superset/databases/schemas.py

```diff
@@ -482,6 +502,8 @@ class DatabaseTestConnectionSchema(Schema, DatabaseParametersSchemaMixin):
         validate=[Length(1, 1024), sqlalchemy_uri_validator],
     )
 
+    ssh_tunnel_credentials = fields.Nested(DatabaseSSHTunnel, allow_none=True)
```

Comment:
Do we need to rename this to be `ssh_tunnel` in this file?

Reply:
+1